gstreamer-sdp/src/m_i_k_e_y_payload_s_p_param.rs

```diff
@@ -29,6 +29,8 @@ impl MIKEYPayloadSPParam {
     }
 
     pub fn val(&self) -> u8 {
-        *self.0.val
+        unsafe {
+            *self.0.val
```

Comment:
This should use `slice::from_raw_parts` and return a `&[u8]`

Reply:
Ah, dammit I still have to get used to the idea of a u8 pointer actually being an array/slice instead of a single u8... t.t